docs/src/ComponentsPage.js

```diff
@@ -57,6 +60,25 @@ const ComponentsPage = React.createClass({
             <div className='row'>
               <div className='col-md-9' role='main'>
 
+                {/* Custom styles */}
+                <div className='bs-docs-section'>
+                  <h1 className='page-header'><Anchor id='custom-styles'>Custom styles</Anchor></h1>
+                  <h2><Anchor id='custom-styles-with-warnings'>With no validation.</Anchor></h2>
+                  <p>If you need to use your own custom style class, you can set it via <code>bsStyle</code> property as well as predefined ones. Like that</p>
+                  <div className="highlight">
+                    <CodeExample
+                      mode="javascript"
+                      codeText={
+`<Button bsStyle='myCustom'>My button</Button>
+<Button bsStyle='mybook'>MyBook button</Button>`
+                      }
+                    />
+                  </div>
+                  <p>But it will warn about invalid <code>bsStyle</code> property</p>
```

Comment:
I still feel a bit weird telling ppl to use something that will throw a warning? maybe we should just hold off on any documentation until all the issues around this are addressed?